build.gradle

```diff
@@ -2,13 +2,66 @@ plugins {
 	id 'fabric-loom' version '0.2.5-SNAPSHOT'
 }
 
+repositories {
+	mavenLocal()
+	maven {
+		name = "CurseForge"
+		url = "https://minecraft.curseforge.com/api/maven"
+	}
+	maven {
+		name = "GitHub"
+		url = "https://maven.pkg.github.com"
+	}
+	maven {
+		name = "JitPack"
+		url = "https://jitpack.io"
+	}
+	maven {
+		name = "NerdHubMC"
+		url = "https://maven.abusedmaster.xyz/"
+	}
+}
+
 sourceCompatibility = 1.8
 targetCompatibility = 1.8
 
 archivesBaseName = "spooky-time"
 version = "1.0.0"
 
 minecraft {
+	
+}
+
+dependencies {
+	minecraft "com.mojang:minecraft:1.14.4"
+	mappings "net.fabricmc:yarn:1.14.4+build.13"
+	modApi "net.fabricmc:fabric-loader:0.6.2+build.166"
+	modApi "net.fabricmc.fabric-api:fabric-api:0.4.0+build.240-1.14"
+  
+    modImplementation "com.github.Draylar:world-traveler:1.0.4"
+
+	modApi ("io.github.prospector:modmenu:1.7.13+build.123") {
+		transitive = false
+	}
+	
+	modCompile "com.github.NerdHubMC.Cardinal-Components-API:cardinal-components-base:${cardinal_components_version}"
+	include "com.github.NerdHubMC.Cardinal-Components-API:cardinal-components-base:${cardinal_components_version}"
+	modCompile "com.github.NerdHubMC.Cardinal-Components-API:cardinal-components-entity:${cardinal_components_version}"
+	include "com.github.NerdHubMC.Cardinal-Components-API:cardinal-components-entity:${cardinal_components_version}"
+	modImplementation "com.github.emilyploszaj:trinkets:${trinkets_version}"
+}
```

Comment:
This already exists lower down in the build.gradle.

Reply:
Reorganized things to not have duplicates